tools/fs/files.ts

```diff
@@ -360,17 +360,10 @@ export const rm_recursive = Profile("files.rm_recursive", async (path: string) =
 export function fileHash(filename: string) {
   const crypto = require('crypto');
   const hash = crypto.createHash('sha256');
-  hash.setEncoding('base64');
-  const rs = createReadStream(filename);
-  return new Promise(function (resolve) {
-    rs.on('end', function () {
-      rs.close();
-      resolve(hash.digest('base64'));
-    });
-    rs.pipe(hash, { end: false });
-  }).await();
+  const fileBuff = readFile(filename);
+  hash.update(fileBuff);
+  return hash.digest('base64');
```

Comment:
I'm still thinking if this one is correct but I was not thinking of making this async